README.md

```diff
@@ -166,543 +119,10 @@ There are two parameters:
-
-	// select tuples by condition ( primay key > 15 ) with offset 7 limit 5
-	// BTREE index supposed
-	resp, err = client.Select(spaceNo, indexNo, 7, 5, tarantool.IterGt, []interface{}{uint(15)})
-    // or
-	resp, err = client.Select("test", "primary", 7, 5, tarantool.IterGt, []interface{}{uint(15)})
-	log.Println("Select")
-	log.Println("Error", err)
-	log.Println("Code", resp.Code)
-	log.Println("Data", resp.Data)
-
-	// call function 'func_name' with arguments
-	resp, err = client.Call("func_name", []interface{}{1, 2, 3})
-	log.Println("Call")
-	log.Println("Error", err)
-	log.Println("Code", resp.Code)
-	log.Println("Data", resp.Data)
-
-	// run raw lua code
-	resp, err = client.Eval("return 1 + 2", []interface{}{})
-	log.Println("Eval")
-	log.Println("Error", err)
-	log.Println("Code", resp.Code)
-	log.Println("Data", resp.Data)
-}
-```
-
-To enable support of UUID in msgpack with [google/uuid](https://github.com/google/uuid),
-import tarantool/uuid submodule.
-```go
-package main
-
-import (
-	"log"
-	"time"
-
-	"github.com/tarantool/go-tarantool"
-	_ "github.com/tarantool/go-tarantool/uuid"
-	"github.com/google/uuid"
-)
-
-func main() {
-	server := "127.0.0.1:3013"
-	opts := tarantool.Opts{
-		Timeout:       500 * time.Millisecond,
-		Reconnect:     1 * time.Second,
-		MaxReconnects: 3,
-		User:          "test",
-		Pass:          "test",
-	}
-	client, err := tarantool.Connect(server, opts)
-	if err != nil {
-		log.Fatalf("Failed to connect: %s", err.Error())
-	}
-
-	spaceNo := uint32(524)
-
-	id, uuidErr := uuid.Parse("c8f0fa1f-da29-438c-a040-393f1126ad39")
-	if uuidErr != nil {
-		log.Fatalf("Failed to prepare uuid: %s", uuidErr)
-	}
-
-	resp, err := client.Replace(spaceNo, []interface{}{ id })
-
-	log.Println("UUID tuple replace")
-	log.Println("Error", err)
-	log.Println("Code", resp.Code)
-	log.Println("Data", resp.Data)
-}
-```
-
-## Schema
-
-```go
-    // save Schema to local variable to avoid races
-    schema := client.Schema
-
-    // access Space objects by name or id
-    space1 := schema.Spaces["some_space"]
-    space2 := schema.SpacesById[20] // it's a map
-    fmt.Printf("Space %d %s %s\n", space1.Id, space1.Name, space1.Engine)
-    fmt.Printf("Space %d %d\n", space1.FieldsCount, space1.Temporary)
-
-    // access index information by name or id
-    index1 := space1.Indexes["some_index"]
-    index2 := space1.IndexesById[2] // it's a map
-    fmt.Printf("Index %d %s\n", index1.Id, index1.Name)
-
-    // access index fields information by index
-    indexField1 := index1.Fields[0] // it's a slice
-    indexField2 := index1.Fields[1] // it's a slice
-    fmt.Printf("IndexFields %s %s\n", indexField1.Name, indexField1.Type)
-
-    // access space fields information by name or id (index)
-    spaceField1 := space.Fields["some_field"]
-    spaceField2 := space.FieldsById[3]
-    fmt.Printf("SpaceField %s %s\n", spaceField1.Name, spaceField1.Type)
-```
-
-## Custom (un)packing and typed selects and function calls
-
```

Comment:
You can specify user-defined packing/unpacking functions for your types. 
This allows you to store complex structures within a tuple and may speed up 
your requests.

Alternatively, you can just instruct the `msgpack` library to encode your
structure as an array. This is safe "magic". It is easier to implement than
a custom packer/unpacker, but it will work slower.

Reply:
Updated.

Reply:
```diff
--- a/example_custom_unpacking_test.go
+++ b/example_custom_unpacking_test.go
@@ -65,11 +65,12 @@ func (c *Tuple2) DecodeMsgpack(d *msgpack.Decoder) error {
 // Example demonstrates how to use custom (un)packing with typed selects and
 // function calls.
 //
-// You can specify custom pack/unpack functions for your types. This will allow
-// you to store complex structures inside a tuple and may speed up you requests.
+// You can specify user-defined packing/unpacking functions for your types.
+// This allows you to store complex structures within a tuple and may speed up
+// your requests.
 //
-// Alternatively, you can just instruct the `msgpack` library to encode your
-// structure as an array. This is safe "magic". It will be easier to implement than
+// Alternatively, you can just instruct the msgpack library to encode your
+// structure as an array. This is safe "magic". It is easier to implement than
 // a custom packer/unpacker, but it will work slower.
 func Example_customUnpacking() {
        // Establish a connection.
```